src/components/core/core.js

```diff
@@ -316,11 +320,11 @@ export default class Core extends UIObject {
   isFullscreen() {
     // Ensure current instance is in fullscreen mode by checking fullscreen element
     const fullscreenElement = Fullscreen.fullscreenElement()
-    const isElementInFullscreen = fullscreenElement && ((fullscreenElement === this.el) 
-    || (fullscreenElement === this.activePlaybackEl)) 
-    || this.activePlaybackEl.webkitDisplayingFullscreen
-    
-    return isElementInFullscreen 
+    const isElementInFullscreen = fullscreenElement && ((fullscreenElement === this.el)
+      || (fullscreenElement === this.activePlaybackEl))
+      || this.activePlaybackEl.webkitDisplayingFullscreen
+
+    return isElementInFullscreen
```

Comment:
```suggestion
    return !!isElementInFullscreen
```

As previously mentioned, we could use type coercion here and evade the use of `toBeFalsy`.